VerySimpleExpressions.lhs

```diff
@@ -199,15 +200,15 @@ and b are numbers.
 > data SingleAdd = SingleAdd Integer Integer
 >                  deriving (Eq,Show)
 >
-> singleAddExamples :: [(String, SingleAdd)]
-> singleAddExamples = [("1+2", SingleAdd 1 2)
->                     ,("101+202", SingleAdd 101 202)]
+> singleAddExamples :: [(String, Either ParseError SingleAdd)]
+> singleAddExamples = [("1+2", Right (SingleAdd 1 2))
+>                     ,("101+202", Right (SingleAdd 101 202))]
 
 > add :: Parser SingleAdd
 > add = do
->     e0 <- many1 digit
->     void $ char '+'
->     e1 <- many1 digit
+>     e0 <- lexeme $ many1 digit
```

Comment:
You made good fixes to these parsers using lexeme, but the tutorial introduces the lexeme parser later, and these parsers are supposed to fail on whitespace issues at this point so we can fix them later.